crates/ra_hir/src/ty/tests.rs

```diff
@@ -3546,11 +3546,11 @@ fn test(x: Trait, y: &Trait) -> u64 {
     [129; 132) 'bar': fn bar() -> {unknown}
     [129; 134) 'bar()': {unknown}
     [140; 141) 'x': {unknown}
-    [140; 147) 'x.foo()': {unknown}
+    [140; 147) 'x.foo()': u64
```

Comment:
we don't resolve bare `Trait` yet, so we just pass a placeholder to Chalk during method resolution, and now it returns "ambiguous" instead of "no" for that query... so as expected